harness/determined/cli/task.py

```diff
@@ -97,7 +96,8 @@ def logs(args: Namespace) -> None:
             timestamp_after=args.timestamp_after,
         )
         if args.json:
-            api.print_json_logs(logs)
+            for log in logs:
+                render.print_json(log.to_json(), skip_coloring=True)
```

Comment:
**suggestion**: there are originally ~3 calls to `api.print_json_logs` , but this PR replaces all of them. Since the function `print_json_logs` can be eliminated entirely, remove the function from `common/api/logging.py` and `common/api/__init__.py` to reduce tech debt

Reply:
ah yeah I missed that. I removed it.